sway-core/src/declaration_engine/declaration_wrapper.rs

```diff
@@ -17,6 +17,7 @@ use crate::{
 pub(crate) enum DeclarationWrapper {
     // no-op variant to fulfill the default trait
     Unknown,
+    Enum(TypedEnumDeclaration),
```

Comment:
There is already a PR for adding enum support: https://github.com/FuelLabs/sway/pull/2713. Please remove these additions on this PR.